app/controllers/OtherReasonController.scala

```diff
@@ -139,6 +139,7 @@ class OtherReasonController @Inject()(whenDidBecomeUnablePage: WhenDidBecomeUnab
       if(request.cookies.get("jsenabled").isDefined) {
         Future(Redirect(controllers.routes.OtherReasonController.onPageLoadForUploadEvidence(mode)))
       } else {
+        val nextPageWithUpload = navigation.nextPage(UploadCompletePage, mode)
```

Comment:
The nextPage takes the current page as a parameter, so FirstFileUploadPage would be more appropriate